### ISSUE TITLE
WAI-ARIA accessibility guidelines for Notebook

### ISSUE THREAD

**Issue by techtonik** [ipython/ipython#7979]
https://en.wikipedia.org/wiki/WAI-ARIA describes a way how to make HTML5 pages friendly for screenreaders. It should be possible to implement these in IPython Notebook.

The topic of accessible IDE for Python was brought up on python-ideas here:
https://groups.google.com/d/topic/python-ideas/DnK6f4trlUk/discussion

**Comment by minrk** [comment]
We should definitely be working on this. Do you know about testing/coverage tools for such things?

**Comment by techtonik** [comment]
@minrk, no. But it would be interesting to research. If there are any funds that can back this up, I know a few people who can help.
